src/fileimpl.cpp

```diff
@@ -166,6 +170,16 @@ offset_t readOffset(const Reader& reader, size_t idx)
 
       current += (len + 1);
     }
+
+    const_cast<bool&>(m_newNamespaceScheme) = header.getMinorVersion() >= 1;
+    if (m_newNamespaceScheme) {
+      const_cast<entry_index_t&>(m_startUserEntry) = getNamespaceBeginOffset('C');
+      const_cast<entry_index_t&>(m_endUserEntry) = getNamespaceEndOffset('C');
+    } else {
+      const_cast<entry_index_t&>(m_endUserEntry) = getCountArticles();
+    }
```

Comment:
Ugly

Reply:
What do you propose ? Simply remove the const ?